docs/tips.md

```diff
@@ -1,9 +1,9 @@
 ## Connect to an FTP server hosted inside XQEMU
 Writing files to your Xbox's HDD is currently a pain. It will be easy once the
 fatx driver has write capabilities, but until then you have the option of
-connecting over FTP to a server hosted inside xqemu.
+connecting over FTP to a server hosted inside XQEMU.
```

Comment:
nit: Capitalization and HTTPS should have been different commits, and maybe even different PRs (as we might reject one of them for some reason). It's fine this time though.